### ISSUE TITLE
data_setter: zero validation set and option for data preprocessing

### ISSUE THREAD

**Issue by kimbob13** [Lee-Gihun/Pytorch-OSI-setup#7]
In data_setter, it would be nice to address one issue and add another feature

1. Issue
It is needed to address case where validation set size is zero.

2. Feature
Make a option for data preprocessing. (i.e. enable or disable preprocessing)
